### PR TITLE
Add basic server client communication

### PR DESCRIPTION
**Pull request by Bananonymous**

### CODE REVIEW

File: dependency-reduced-pom.xml
Comment (Syseria) [inline dependency-reduced-pom.xml:1]: We must keep spotless.

**Review by Syseria — CHANGES_REQUESTED** [review]
Apart from spotless, rest is ok.

File: dependency-reduced-pom.xml
Comment (Bananonymous) [reply dependency-reduced-pom.xml:1]: Added spotless

**Review by Syseria — APPROVED** [review]
Spotless !

**Comment by Bananonymous** [comment]
Closes #1

**Pull request merged**